ckanext/blob_storage/tests/test_blob_storage_bug_fix.py

```diff
@@ -0,0 +1,85 @@
+"""Tests for plugin.py."""
+# encoding: utf-8
+from ckan.plugins import toolkit
+from ckan.tests import factories
+import pytest
+import logging
+from ckanext.unaids.tests import get_context
+import mock
+
+
+log = logging.getLogger(__name__)
+
+
+@pytest.mark.usefixtures('clean_db', 'with_plugins')
+class TestBlobStorageActivityDownload(object):
+    def test_can_download_release_resource_whether_it_exists_in_current_version_of_package_or_not(self, app, org_admin):
+
+        org = factories.Organization(user=org_admin)
+        context = get_context(org_admin)
```

Comment:
I don't think you need the org_admin context here. It's only used in test setup ckan actions call.